mvds/mdf.md

```diff
@@ -0,0 +1,79 @@
+#  MVDS Meta Data Field
+
+> Version: 0.1.0 (Draft)
+> 
+> Authors: Oskar Thorén <oskar@status.im>, Dean Eigenmann <dean@status.im>
+
+##  Table of Contents
+
+1. [Abstract](#abstract)
+2. [Format](#format)
+    1. [Fields](#fields) 
+1. [Usage](#usage)
+    1. [Information](#information)
+    2. [Behavior](#behavior)
+
+## Abstract
+
+In this specification, we describe a method to construct a message DAG (Directed Acyclic Graph) that will aid the consistency of [MVDS](./README.md). Additionally we explain how data sync can be used for more lightweight messages that do not require full synchronization. This specification extends the [MVDS message](./README.md#payloads) to modify the functionality of MVDS.
```

Comment:
This spec should support both linear and DAG history. I.e. both sequence numbers in a log (appended to by one keypair, or some tbd light consensus) as well as DAG with. Right now it seems like it is purely about DAG